ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -471,6 +472,23 @@ - (NSString *)localizedInvalidValueStringWithAnswerString:(NSString *)text {
     return nil;
 }
 
+- (NSString *)stringForAnswer:(id)answer {
+    ORKAnswerFormat *impliedFormat = [self impliedAnswerFormat];
+    return impliedFormat == self ? nil : [impliedFormat stringForAnswer:answer];
+}
+
+- (NSString*)stringForChoiceAnswer:(id)answer {
+    __block NSMutableArray<NSString *> *answerStrings = [[NSMutableArray alloc] init];
+    ORKChoiceAnswerFormatHelper *helper = [[ORKChoiceAnswerFormatHelper alloc] initWithAnswerFormat:self];
+    for (NSNumber *index in [helper selectedIndexesForAnswer:answer]) {
+        NSString *text = [[helper answerOptionAtIndex:[index integerValue]] text];
+        if (text != nil) {
+            [answerStrings addObject:text];
+        }
+    }
+    return [answerStrings componentsJoinedByString:@"\n"];
```

Comment:
It's not quite safe to place `stringForChoiceAnswer` in `ORKAnswerFormat` class, because only some of its subclass comply to `ORKChoiceAnswerFormatHelper`'s initiator. How about we move this method entirely to  `ORKChoiceAnswerFormatHelper`?